css/tablesort.css

```
@@ -0,0 +1,26 @@
+th[role=columnheader]:not(.no-sort) {
```

```suggestion
table[data-sortable-table] th[role=columnheader]:not(.no-sort) {
```
should we limit the CSS code to sortable tables?
Otherwise every `th[role=columnheader]` would get styled which is probably not what we want.

---

Added this in https://github.com/contao-components/tristen-tablesort/pull/1/commits/2ebc616fa2675b52b6a3b92f4765814298af658f